build-system/global-configs/canary-config.json

```diff
@@ -33,5 +33,6 @@
   "expUnconditionedAdxIdentity": 0.01,
   "expUnconditionedDfpIdentity": 0.01,
   "expUnconditionedCanonicalHoldback": 0.01,
-  "amp-consent": 1
+  "amp-consent": 1,
+  "amp-img-native-srcset": 1
```

Comment:
/cc @erwinmombay Does Canary config differing from Prod config actually work? ( I don't remember ever doing this in the past. It does make sense for us to do this here, just want to make sure it actually does something)

Reply:
https://github.com/ampproject/amphtml/blob/89b69a952b27c091e2a594a297a73c1afcbcf800/build-system/global-configs/README.md

I think per this documentation, `canary-config.js` is actually just 1% of production, so for 1%, we can just opt in all of canary and leave production alone.

Reply:
yeah Canary always goes to 1% of production on or around Wednesdays anyway. it is fine to keep what you have. May endup being ~1.5%